test/spec/app_spec.rb

```diff
@@ -779,3 +788,4 @@
     # end
   end
 end
+end
```

Comment:
Possibly a better pattern than a lambda, but wrapping rspec blocks in a class seemed more complex so I'm starting here.

Also, we can indent once we like this; it screwed up the diff though.